data/filter.go

```diff
@@ -105,7 +101,7 @@ var (
 	Datasets = ContentType{
 		LocaliseKeyName: "Datasets",
 		Type:            "datasets",
-		SubTypes:        []string{"dataset_landing_page", "reference_tables"},
+		SubTypes:        []string{"dataset_landing_page", "timeseries_dataset", "reference_tables"},
```

Comment:
As you removed `reference_tables` from `defaultContentTypes`, you need to remove it from here as well. This is in the assumption that we don't consider search results with the type `reference_tables` or they just don't exist